yb-voyager/src/query/queryissue/issues_dml_test.go

```diff
@@ -59,6 +58,29 @@ func testRegexFunctionsIssue(t *testing.T) {
 	}
 }
 
+func testJsonConstructorFunctions(t *testing.T) {
+	ctx := context.Background()
+	conn, err := getConn()
+	assert.NoError(t, err)
+
+	defer conn.Close(context.Background())
+	_, err = conn.Exec(ctx, `select json_object('code' VALUE 'P123', 'title': 'Jaws');`)
```

Comment:
can we check for each json construct and query functions here. Since we don't know how YB will start supporting them - either one by one or all at once.